src/rill/event_store/psql.clj

```diff
@@ -20,14 +20,8 @@
                 m)
         stamped (if-let [timestamp ^Timestamp (:created_at r)]
                   (assoc! typed message/timestamp (Date. (.getTime timestamp)))
-                  typed)
-        id-d (if-let [id (:event_id r)]
-               (assoc! stamped message/id (uuid id))
-               stamped)
-        streamed (if-let [stream-id (:stream_id r)]
```

Comment:
The stream-id bit here is actually necessary here, because that's how we relate the message to the right stream (topic) when fetching all events (from all streams at once).

The data isn't in the payload (see `strip-metadata` below).

This is also probably the reason the tests fail.

Reply:
ok! But the stream id is already being saved as id + type, so I think there is no need to save it also as event id?